intake_geopandas/geopandas.py

```diff
@@ -70,15 +76,36 @@ def __init__(self, urlpath, bbox=None,
         self._bbox = bbox
         self._geopandas_kwargs = geopandas_kwargs or {}
         self._dataframe = None
+        self.storage_options = storage_options or {}
 
         super().__init__(metadata=metadata)
 
     def _open_dataset(self):
         """
         Open dataset using geopandas and use pattern fields to set new columns.
         """
+        # new caching
+        #url = fsspec.open_local(self.urlpath, **self.storage_options)
+        
+        # old caching
+        if self.cache:
+            print('inside old caching')
+            self.cache[0].load(self.urlpath)
+            #import os
+            #assert os.listdir(self.cache[0]._path(self.urlpath))
+        
+        if self.cache:
+            try:
+                url = 'zip://'+self.cache[0]._path(self.urlpath)
```

Comment:
I needed to add `zip://` here. but this might not be needed everywhere? For now, just a first draft. maybe we need to add here some more detail to make it work for `s3` and other fs

Reply:
@ian-r-rose do you have an example of a shapefile on `s3` for testing?

Reply:
Hmm, I don't have one off hand, but you might poke around the AWS public dataset program, e.g., [this](https://s3.console.aws.amazon.com/s3/buckets/njogis-elevation/?region=us-west-2&tab=overview) bucket.

Otherwise (and it looks like you are already doing this), tests using HTTP seem like a good starting point.